core/src/main/resources/reference.conf

```diff
@@ -407,6 +406,21 @@ cassandra-query-journal {
     # before the stream starts emitting events.
     offset-scanning-period = 200ms
 
+
+    back-track {
+      # Interval at which events by tag stages trigger a query to look for delayed events from before the
+      # current offset. Delayed events are also searched for when a subsequent event for the same persistence id
+      # is received. The purpose of this back track is to find delayed events for persistence ids that are no
+      # longer receiving events. Depending on the size of the period of the back track this can be expensive.
+      interval = 1s
+
+      # How far back to go in time for the scan. This is the maxinum amount of time an event can be delayed
+      # and be found without receiving a new event for the same persistence id. Set to a duration or "max" to search back
+      # to the start of the previous bucket which is the furthest a search can go back.
+      period = 5s
```

Comment:
what's our thinking about the low frequency scan further back vs. the more frequent short scan?

Reply:
a rather likely scenario would be that a tag write fails, and is retried later (more than 5s later) and then it will not show up in the eventsByTag

Reply:
Perhaps initially a larger default than 5s? 30s? 

The short/long poll would be straightforward so alternatively we could just do that now

Reply:
I'd sleep better if we also had an infrequent 2 x bucket scan

Reply:
I'll add that now

Reply:
With the caveat it should not go further back than the persistence id cleanup we did, otherwise it'll re-deliver events not knowing it had already done them